src/recipes/paloma.json

```
@@ -0,0 +1,25 @@
+{
+    "name": "Paloma",
+    "description": "A simple tequila cocktail made with grapefruit soda",
+    "github": "estudioChispa",
+    "ingredients": [
+        "2 or 3 oz tequila (I prefer a silver or blanco, but use what you like)",
+        "Grapefruite soda (like Fresca, Squirt, or Jarritos)",
+        "2 Limes",
+        "Salt",
+        "(optional) Tajin - a seasoning mix of lime, chilies, and salt ",
+        "Ice"
+    ],
+    "directions": [
+        "Rim your glass with a wedge of lime and the Tajin (if using) or salt",
+        "Squeeze the other lime into the glass"
```

Missing comma, breaking json.
```suggestion
        "Squeeze the other lime into the glass",
```

---

Dang it! Done (sorry)